scripts/dog-info.js

```diff
@@ -0,0 +1,108 @@
+    $(this).css('box-shadow', 'none')
+  })
+
+
+  /*********************************CHECKOUT**************************************/
+
+  $('form').submit(function(event) {
+    event.preventDefault();
+    alert('Thank you. The form information has been received.');
+    console.log($(this).serializeArray());
+  });
+
+
+  /********************************BLOG***************************************/
+
+  let post;
+
+  const blogArticles = [
+    {
+      id: 'blog1',
+      title: 'Traveling With Your Dog',
+      fullText: 'Pellentesque tellus velit, finibus ac nisi non, euismod aliquam magna. In ac pretium ante, sit amet iaculis ligula. Proin viverra accumsan est, ut semper sapien imperdiet non. Pellentesque blandit accumsan sollicitudin. Nulla scelerisque facilisis semper. Fusce aliquet, turpis vitae sodales porta, sapien magna mattis dui, ut hendrerit risus lectus et ligula. Nullam eu nunc ultrices, mattis purus non, sollicitudin lacus. Curabitur varius id leo vel vulputate. Aenean non elit eleifend, laoreet sapien eget, vestibulum ipsum. Suspendisse ac ex ut mauris sodales accumsan.',
+      imageURL: 'images/blog-1.jpg',
+      imageAltText: 'Woman with dog sitting on a cliff overlooking a valley'
+    },
+    {
+      id: 'blog2',
+      title: 'How to Walk Multiple Dogs',
+      fullText: 'Pellentesque tellus velit, finibus ac nisi non, euismod aliquam magna. In ac pretium ante, sit amet iaculis ligula. Proin viverra accumsan est, ut semper sapien imperdiet non. Pellentesque blandit accumsan sollicitudin. Nulla scelerisque facilisis semper. Fusce aliquet, turpis vitae sodales porta, sapien magna mattis dui, ut hendrerit risus lectus et ligula. Nullam eu nunc ultrices, mattis purus non, sollicitudin lacus. Curabitur varius id leo vel vulputate. Aenean non elit eleifend, laoreet sapien eget, vestibulum ipsum. Suspendisse ac ex ut mauris sodales accumsan.',
+      imageURL: 'images/blog-2.jpg',
+      imageAltText: 'Four dogs on a leash, sitting down in a park, one of them is yawning'
+    },
+    {
+      id: 'blog3',
+      title: 'Teach Your Dog to Fetch!',
+      fullText: 'Pellentesque tellus velit, finibus ac nisi non, euismod aliquam magna. In ac pretium ante, sit amet iaculis ligula. Proin viverra accumsan est, ut semper sapien imperdiet non. Pellentesque blandit accumsan sollicitudin. Nulla scelerisque facilisis semper. Fusce aliquet, turpis vitae sodales porta, sapien magna mattis dui, ut hendrerit risus lectus et ligula. Nullam eu nunc ultrices, mattis purus non, sollicitudin lacus. Curabitur varius id leo vel vulputate. Aenean non elit eleifend, laoreet sapien eget, vestibulum ipsum. Suspendisse ac ex ut mauris sodales accumsan.',
+      imageURL: 'images/blog-3.jpg',
+      imageAltText: 'Silhouettes of person playing fetch with dog at sunset'
+    }
+  ];
+
+  for (let i = 0; i < blogArticles.length; i += 1) {
+    let blogArticle = blogArticles[i];
+    post = document.getElementById(blogArticle.id);
```

Comment:
While this runs great, one thing to think about is how this implementation might become difficult if you are needing to dynamically generate lots of blog articles. You'd have to keep track of how many articles and continually add div elements on your html page. The solution is to have one div on your blog page, maybe call it `article-container` then using either jQuery or plain HTML you'd dynamically add your articles to this container as your loop progresses. You can look at using appendChild() using documentation here: https://www.w3schools.com/jsref/met_node_appendchild.asp